.eslintrc.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
             files: ['*.js', '*.jsx', '*.ts', '*.tsx'],
             rules: {
                 'rulesdir/onyx-props-must-have-default': 'off',
+                'rulesdir/no-multiple-onyx-in-file': 'error',
```

Comment:
This is not necessary. We already define the rule in `eslint-config-expensify/rules/expensify`.

Reply:
Noted, I've checked after remove it and It's working as expected